dedibox-network/rpn/reference-content/private-networks-overview.mdx

```diff
@@ -50,7 +50,7 @@ In practice, let us assume that you have six dedicated servers. You can create R
 ​
 It is also possible to add servers from other Scaleway Dedibox accounts to the RPN, if their owner gives permission to do so.
 ​
-With the **RPNv2** feature, available on our best-selling Dedibox servers, you get a real VLAN for your private network and up to 25GBit/s of available bandwidth, making it an ideal solution for latency-critical and bandwidth-intense applications such as network attached storage, large databases or load balancing. You can configure any network setup you have in mind using the RPNv2 network and even use it for communications between your virtual machines on several Dedibox servers. We also have a gateway for communications between RPNv1 and RPNv2 networks, enabling the two types to "talk" to each other.
+With the **RPNv2** feature, available on our best-selling Dedibox servers, you get a real VLAN for your Private Network and up to 25GBit/s of available bandwidth, making it an ideal solution for latency-critical and bandwidth-intense applications such as network attached storage, large databases or load balancing. You can configure any network setup you have in mind using the RPNv2 network and even use it for communications between your virtual machines on several Dedibox servers. We also have a gateway for communications between RPNv1 and RPNv2 networks, enabling the two types to "talk" to each other.
```

Comment:
I think here it isn't a product name (only for Elements)

Reply:
```suggestion
With the **RPNv2** feature, available on our best-selling Dedibox servers, you get a real VLAN for your private network and up to 25GBit/s of available bandwidth, making it an ideal solution for latency-critical and bandwidth-intense applications such as network-attached storage, large databases, or load balancing. You can configure any network setup you have in mind using the RPNv2 network and even use it for communications between your virtual machines on several Dedibox servers. We also have a gateway for communications between RPNv1 and RPNv2 networks, enabling the two types to "talk" to each other.
```